buildSrc/src/main/kotlin/plugin/DependencyHandleScopeExt.kt

```diff
@@ -0,0 +1,23 @@
+package plugin
+
+import org.gradle.api.artifacts.dsl.DependencyHandler
+
+fun DependencyHandler.addImplementation(notation: String) {
```

Comment:
@wasinpp hi mas thank you for the PR :) 

one question, afaik kotlin kts already has this no?

Reply:
That is my first try too.
But it seems there is no such implementation in DependencyHandler.
Let me know if there are alternative approach.

Reply:
so in gradle kotlin in class `ImplementationConfigurationAccesssors` it has extension functions for these declarations

Reply:
take a look at [wikipedia/build.gradle](https://github.com/radityagumay/wikipedia-android/blob/master/app/build.gradle.kts#L74 )internal

Reply:
That extension is used in the Plugin. Since on the plugin it cannot reference `ImplementationConfigurationAccesssors`. 
https://github.com/radityagumay/QualityMatters/pull/1/files#diff-ad6d7ad957755336ec7bd378b334cc7b82353ce580585634e62c9c02a63696a2R35
Maybe I misconfigure project somewhere, but still don't find any misconfiguration yet.

It is similar to this issue: https://github.com/gradle/gradle/issues/12737
But trying to rename the plugin file that above issue mention doesn't solve.

Reply:
hmm, never face that issue.

have you give a try this mas? https://github.com/gradle/gradle/issues/12737#issuecomment-611670621

Reply:
@wasinpp hi any updates on this?

Reply:
Just successfully do it as the precompile plugin, but I just raise different problem.
https://github.com/radityagumay/QualityMatters/pull/1/commits/90b65f6e05ccaa8c97193a27e59500dd95715131

With the precompile plugin approach it need to specify either as the com.android.application or android-library on the plugins dsl.
<img width="345" alt="Screen Shot 2020-12-04 at 08 39 52" src="https://user-images.githubusercontent.com/45651229/101110766-48927f80-360c-11eb-8e74-d3f630a190cd.png">
This make the plugin only used by either application or android-library. 

So I dont think this precompile plugin approach is suitable for this case mas.